mage_ai/__init__.py

```diff
@@ -112,3 +115,11 @@ def clean(
     else:
         _, df_clean = clean_df(df, name=name, verbose=verbose)
     return df_clean
+
+# Data preparation methods
```

Comment:
Maybe make the comment more visible like
`# ------------------------- Data preparation methods ------------------------- #`